raiden/tests/unit/transfer/mediated_transfer/test_targetstate.py

```diff
@@ -260,34 +250,20 @@ def test_handle_offchain_secretreveal_after_lock_expired():
 
     Target part for https://github.com/raiden-network/raiden/issues/3086
     """
-    amount = 3
-    block_number = 1
-    expiration = block_number + factories.UNIT_REVEAL_TIMEOUT
-    initiator = factories.HOP1
-    our_address = factories.ADDR
-    secret = factories.UNIT_SECRET
-    pseudo_random_generator = random.Random()
-
-    channel_state, state = make_target_state(
-        our_address,
-        amount,
-        block_number,
-        initiator,
-        expiration,
-    )
+    setup = make_target_state()
 
-    lock_expiration = state.transfer.lock.expiration
+    lock_expiration = setup.new_state.transfer.lock.expiration
     lock_expiration_block_number = lock_expiration + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS * 2
     lock_expiration_block = Block(
         block_number=lock_expiration_block_number,
         gas_limit=1,
         block_hash=factories.make_transaction_hash(),
     )
     iteration = target.state_transition(
-        target_state=state,
+        target_state=(setup.new_state),
```

Comment:
Why is this in parentheses?

Reply:
Used a PyCharm refactoring function here that does this sometimes. I removed all the superfluous parantheses.